routes/index.js

```diff
@@ -39,6 +39,56 @@ router.get('/', function(req, res) {
 	}
 });
 
+/* Group by ad networks display. */
+router.get('/groupBy', function(req,res){
+	try{
+		MongoClient.connect(url, function(err, db){
+			var benchmarkDB = db.collection('benchmark_logs');
+			try{
+				benchmarkDB.aggregate(
+					[
+						{
+							$group: {
+								_id : "$adNetworkUrl",
+								avgLoadTime : { $avg : "$assetCompleteTime"}
+							}
+						},
+						{
+							$sort: {
+								avgLoadTime: -1
+							}
+						},
+						{
+							$project:
+							{
+								_id: "$_id",
+								avgLoadTime:
+								{
+									$divide:[
+										{$subtract:[
+											{$multiply:['$avgLoadTime',1000]},
+											{$mod:[{$multiply:['$avgLoadTime',1000]}, 1]}
+										]},
+										1000
+									]
+								}
+							}
+						}
+					]
+				).toArray(function (err, avgLoadTimes){
+					benchmarkDB.find().count(function (err, total) {
+						res.render('groupByAdNets.html', { records : avgLoadTimes.slice(0,99)});
```

Comment:
Probably should name this html file the same as the above route, `groupByAdHost`